pullreq_pipeline.go

```diff
@@ -80,7 +80,7 @@ func deleteStaleGitlabPRBranch(log *logrus.Entry, pr *github.PullRequestEvent, c
 		return nil
 	}
 	repoName := pr.GetRepo().GetName()
-	repoOrg := pr.GetRepo().GetOrganization().GetLogin()
+	repoOrg := *pr.Organization.Login
```

Comment:
This is super strange, I believe the difference here is that the removed line get the login of the organization of the repo, while the second one gets the login of the organization of the pr. Have you tried:

`pr.GetOrganization().GetLogin()`

?

Reply:
My bad, I haven't. See related comment above.